src/web/routes/application/register-form-routes.js

```diff
@@ -29,6 +29,7 @@ const createRoute = (config, csrfProtection, steps, router) => (step) =>
       configureGet(steps, step),
       getSessionDetails,
       handleRequestForPath(config, steps, step),
+      handleOptionalMiddleware(step.behaviour),
```

Comment:
I wonder if this needs some reference to the request method? In the future we will need something similar for address lookup, but this will happen on `POST` rather than `GET`.

Reply:
I'm not sure what you mean, should this be calling a `handleOptionalGetMiddleware` method, is that what you mean, or do you mean it should be `step.getBehaviour`? If the latter, we'll have to be careful with the naming as this just looks like a getter.

Reply:
I meant `step.getBehaviour`. Maybe it can stay as it is for now and we can reconsider when the time comes to implement something similar for `POST` requests.

Reply:
`step.behaviourForGet`?